README.md

```
@@ -1,2 +1,29 @@
+
+Navigate to `http://localhost:3000/`
+
+## Features
+- Real time web-based text chat
+- Allows users to log into the chatroom with a username (user info stored in browser session data)
+- Chat room members will be notified when a user connects/disconnects
+- Scrollable chat feed with timestamps
+
+Example:
+
+![](evan_chat_example.gif)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)